src/css/content-to-slotted-usages.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {Document, ParsedCssDocument, Replacement, Severity, Warning} from 'polymer-analyzer';
+import * as shady from 'shady-css-parser';
+
+import {CssRule} from '../css/rule';
+import {registry} from '../registry';
+import {stripIndentation} from '../util';
+
+class ContentToSlottedUsages extends CssRule {
+  code = 'content-to-slotted-usages';
```

Comment:
How about `'content-selector-to-slotted'`. 

The `usages` isn't necessary, because there isn't another form (we have two passes for `<content>`, one that gets declarations of `<content>` and the other that gets at places where elements expected to be distributed into a `<content>`).